src/main/java/Duke.java

```diff
@@ -1,10 +1,82 @@
+import java.util.Scanner;
+
 public class Duke {
+    static String lineCutOff = "_______________________";
+    static int TODO_TASK_INDEX = 5;
+    static int DEADLINE_TASK_INDEX = 9;
+    static int EVENT_TASK_INDEX = 6;
+
+    public static Task taskType(String task) {
+        if (task.startsWith("todo")){
+            return new ToDo(task.substring(TODO_TASK_INDEX ));
+        }else if (task.startsWith("deadline")){
+            int indexBy = task.indexOf("/");
+            return new Deadline(task.substring(DEADLINE_TASK_INDEX,indexBy - 1), task.substring(indexBy + 3));
+        }else if (task.startsWith("event")){
+            int indexAt = task.indexOf("/");
+            return new Event(task.substring(EVENT_TASK_INDEX, indexAt - 1), task.substring(indexAt + 3));
+        }
+        return null;
+    }
+
+    public static void printTask(Task task, int listNum) {
+        System.out.println(lineCutOff + "\n" + "Got it. I've added this task: ");
+        System.out.println("  " + task.toString());
+        System.out.println("Now you have " + (listNum + 1) + " tasks in the list.");
+        System.out.println(lineCutOff + "\n");
+    }
+
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        //String lineCutOff = "_______________________";
```

Comment:
Maybe this comment and similarly line 40 can be removed, since they do not really explain the context.